nrt/data/DynaFlow/GeneratorPVTfo/GeneratorPVTfo.crv

```diff
@@ -46,6 +46,7 @@
   <curve model="Generator5" variable="reactivePowerControlLoop_level"/>
   <curve model="Generator5" variable="reactivePowerControlLoop_QStatorPu"/>
   <curve model="Generator5" variable="reactivePowerControlLoop_UStatorRefPu"/>
+  <curve model="Generator5" variable="reactivePowerControlLoop_limiter_UStatorMinMaxPu_uMax"/>
```

Comment:
I think you forgot to remove it from the crv, no ?
If you let it here, the curve will be calculated by the nrt and compared to the reference, which has not been updated, so it will fail. You can either remove this line, or update the reference csv, thanks